worlds/yoshisisland/Rules.py

```diff
@@ -0,0 +1,789 @@
+from BaseClasses import MultiWorld
+from .level_logic import YoshiLogic
+from ..generic.Rules import set_rule

```

Comment:
```suggestion
from worlds.generic.Rules import set_rule
```
Relative import from outside an apworld breaks in python 3.8